Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -467,9 +467,9 @@ nameStmt : NAME whiteSpace expression whiteSpace AS whiteSpace expression;
 // 5.4.2.10 Select Case Statement
 selectCaseStmt :
     SELECT whiteSpace? CASE whiteSpace? selectExpression endOfStatement
-    caseClause*
-    caseElseClause?
-    END_SELECT
+    (statementLabelDefinition? whiteSpace? caseClause)*
+    statementLabelDefinition? whiteSpace? caseElseClause?
```

Comment:
Does thin interfere with the quickfixes for the empty if block and empty else block inspections?